docstamp/cli/cli.py

```diff
@@ -57,10 +57,12 @@ def cli():
                    'document from. Note that the samples numbers start from 0 '
                    'and the empty ones do not count.')
 @click.option('--dpi', type=int, default=150, help='Output file resolution')
-@click.option('-g', '--google', is_flag=True, flag_value=True,
+@click.option('-g', '--google', is_flag=True, 
               help='Fetch data from Google Drive')
-@click.option('-v', '--verbose', is_flag=True, flag_value=True,
+@click.option('-v', '--verbose', is_flag=True, 
               help='Output debug logs.')
+@click.option('-u', '--unicode_support', is_flag=True, default=False,
+              help='Allows unicode characters to be correctly encoded in the PDF.')
 def create(input, template, field, outdir, prefix, otype, command, index,
```

Comment:
Could you please add `unicode_support` as a parameter to the `create` function? :)

Reply:
done in ba97c53